abnormal_trajectory_v1_angle.py

```
@@ -0,0 +1,1042 @@
+# -----------------------------
+# UTILITY FUNCTIONS (From original code)
+# -----------------------------
+def extract_trajectory(csv_path: str) -> Optional[np.ndarray]:
+    try:
+        df = pd.read_csv(csv_path, usecols=["frameNo", "left", "top", "w", "h"])
+        df["center_x"] = df["left"] + df["w"] / 2
+        df["center_y"] = df["top"] + df["h"] / 2
+        if len(df) < 2:
+            return None
+        return df[["frameNo", "center_x", "center_y"]].values
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)